lib/wskprops.js

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+// based on serverless-openwhisk, MIT licensed
+// but changed to drop usage of async Promises and some renaming
+// https://github.com/serverless/serverless-openwhisk/blob/master/provider/credentials.js
+'use strict'
+
+const path = require('path')
+const fse = require('fs-extra')
+const process = require('process')
+
+const ENV_PARAMS = ['OW_APIHOST', 'OW_AUTH', 'OW_NAMESPACE', 'OW_APIGW_ACCESS_TOKEN']
```

Comment:
are these standard somewhere? the runtime plugin uses these because the wsk cli uses these:  https://github.com/adobe/aio-cli-plugin-runtime/blob/3456221a2e55cab75c39b08891a24f3cae2c69c3/src/properties.js#L28-L31

Reply:
although I'm now concerned with "cross-contamination" of environment variables (like then env vars I suggested) which will cause side effects with the runtime plugin now... if this is to affect only the app scripts it should be appropriately namespaced. I think this warrants further discussion cc @purplecabbage

Reply:
Also `OW_*` is used by the serverless OpenWhisk plugin which I am not sure we want to support.
The wsk tooling uses `WHISK_*` and `WSK_CONFIG_FILE`. 

+1 to not repeat work in app-scripts and rely on the runtime plugin to parse those env variables, maybe we can expose something in runtime-helpers?

Reply:
@shazron @moritzraho What bothers me is that we create a dependency to runtime-plugin. If we need runtime plugin to run this, runtime is not really a plugin anymore. It should become a core component.

Reply:
@tmathern agree but the runtime plugin is already a dependency, it is imported in app-scripts as if it were a lib via https://github.com/adobe/aio-cli-plugin-runtime/blob/master/src/runtime-helpers.js (which is exposed in package.json). This is temporary and we have some plans to eventually split the runtime-helpers into its own core-library.
But for the time being I would suggest to reuse https://github.com/adobe/aio-cli-plugin-runtime/blob/master/src/properties.js by exposing it somehow in the runtime-helpers.
wdyt @purplecabbage ?